content/blog/spatialsample-0-2-0/index.Rmd

```diff
@@ -0,0 +1,205 @@
+tags: [spatialsample, rsample, tidymodels]
+---
+
+<!--
+TODO:
+* [x] Look over / edit the post's title in the yaml
+* [x] Edit (or delete) the description; note this appears in the Twitter card
+* [x] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [x] Find photo & update yaml metadata
+* [x] Create `thumbnail-sq.jpg`; height and width should be equal
+* [x] Create `thumbnail-wd.jpg`; width should be >5x height
+* [x] `hugodown::use_tidy_thumbnails()`
+* [x] Add intro sentence, e.g. the standard tagline for the package
+* [x] `usethis::use_tidy_thanks()`
+-->
+
+```{r setup, include = FALSE}
+library(ggplot2)
+theme_set(theme_minimal())
+```
+
+
+We're positively electrified to announce the release of [spatialsample](https://spatialsample.tidymodels.org/) 0.2.0. spatialsample is a package for spatial resampling, extending the rsample framework for resampling to help create spatial extrapolation between your analysis and assessment data sets.
+
+You can install it from CRAN with:
+
+```{r, eval = FALSE}
+install.packages("spatialsample")
+```
+
+This blog post will describe the highlights of what's new. You can see a full list of changes in the [release notes](https://github.com/tidymodels/spatialsample/blob/main/NEWS.md).
+
+## New Features
+
+This version of spatialsample includes a new data set, made up of 682 hexagons containing data about tree canopy cover change in Boston, Massachusetts:
+
+```{r}
+#| fig.alt = "A map showing the spatial arrangement of hexagons making up the boston_canopy data set."
+library(spatialsample)
+library(ggplot2)
+
+ggplot(boston_canopy) + 
+  geom_sf()
+```
+
+This data is stored as an sf object, and as such contains information about the proper coordinate reference system and units of measurement associated with the data. 
+
+And that brings us to the first new feature in this release of spatialsample: `spatial_clustering_cv()` now supports sf objects, and will calculate distances in a way that respects coordinate reference systems (including using the s2 geometry library for geographic coordinate reference systems):
+
+```{r}
+set.seed(123)
+kmeans_clustering <- spatial_clustering_cv(boston_canopy, v = 5)
+kmeans_clustering
+```
+
+This release also provides `autoplot()` methods to visualize resamples via ggplot2, making it easy to see how exactly your data is being divided. Just call `autoplot()` on the outputs from any spatial clustering function:
+
+```{r}
+#| fig.alt = "A map showing the boston_canopy data set broken into five folds through spatial_clustering_cv. The five folds are visibly different sizes, and are grouped by spatial proximity."
+autoplot(kmeans_clustering) + labs(title = "kmeans()")
+```
+
+In addition to supporting more types of data, `spatial_clustering_cv()` has also been extended to support more types of clustering. Use the `cluster_function` argument to use hierarchical clustering via `hclust()` instead of the default `kmeans()`-based clusters:
```

Comment:
```suggestion
In addition to supporting more types of data, `spatial_clustering_cv()` has also been extended to support more types of clustering. Set the `cluster_function` argument to `"hclust"` for hierarchical clustering via `hclust()` instead of the default `kmeans()`-based clusters:
```